core/domain/skill_services.py

```diff
@@ -246,6 +246,24 @@ def get_skill_summary_by_id(skill_id, strict=True):
         return None
 
 
+def get_skill_descriptions_by_ids(skill_ids):
+    """Returns a list of skill descriptions corresponding to given skill ids.
+
+    Args:
+        skill_ids: str. The list of skill ids.
+
+    Returns:
+        list(str). The skill descriptions of skills corresponding to given ids.
+    """
+    skill_summary_models = skill_models.SkillSummaryModel.get_multi(skill_ids)
+    skill_descriptions = [
+        skill_summary_model.description
+        for skill_summary_model in skill_summary_models
+        if skill_summary_model is not None
```

Comment:
Um. If a model is None then you're going to get a list with a different number of elements to the original. That seems ... bad? I don't see how the caller is going to figure out which skill corresponds with which description in that case.

This case also needs to have backend tests.

Reply:
Done, changed it to use a dict keyed by skill id, so of a skill doesn't exist, it would just return undefined. Is a backend test needed in this method, as I am not sure if there is anything to test. I think maybe frontend tests would be required to check undefined?